pkg/daemon/update.go

```diff
@@ -121,17 +129,54 @@ func (dn *Daemon) reconcilable(oldConfig, newConfig *mcfgv1.MachineConfig) (bool
 	if !reflect.DeepEqual(oldIgn.Networkd, newIgn.Networkd) {
 		glog.Warningf("daemon can't reconcile state!")
 		glog.Warningf("Ignition networkd section contains changes")
-		return false, nil
+		return false, nil, nil
 	}
 
 	// Passwd section
 
 	// we don't currently configure groups or users in place. we can't fix it if
 	// something changed here.
 	if !reflect.DeepEqual(oldIgn.Passwd, newIgn.Passwd) {
-		glog.Warningf("daemon can't reconcile state!")
-		glog.Warningf("Ignition passwd section contains changes")
-		return false, nil
+		// MCD will handle only ignition SSH keys
+		// This section will check (using some tbd magic) to see if the ONLY diff between 2
+		// Passwd configs is an SSHAuthorizedKey change
+		// Any change other than an SSHChanage will return false.
+
+		sshReconcilable := false
+		// If the size of list Users is different, a User has been added or deleted -> nope
+		if len(oldIgn.Passwd.Users) == len(newIgn.Passwd.Users) {
```

Comment:
Probably worth splitting this into a separate `sshConfigsReconcilable()` or so.

Reply:
This is subject to change once my other PR lands.